src/policy/policy.cpp

```diff
@@ -154,6 +154,60 @@ bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
     return true;
 }
 
+bool IsWitnessStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
+{
+    if (tx.IsCoinBase())
+        return true; // Coinbases are skipped
+
+    for (unsigned int i = 0; i < tx.vin.size(); i++)
+    {
+        // We don't care if witness for this input is empty, since it must not be bloated.
+        // If the script is invalid without witness, it would be caught sooner or later during validation.
+        if (!tx.wit.vtxinwit[i].IsNull()) {
```

Comment:
nit: would prefer early exit of this iteration with `continue;` it makes review easier by not having to check what happen if it is indeed null.